src/utils/repoFilters.js

```diff
@@ -0,0 +1,15 @@
+import moment from 'moment';
+
+export const filterRepos = (type) => (repos) => {
```

Comment:
is there a specific reason we do this currying logic? can we just do

```
export const filterRepos = (type, repos) => {

}
```

Reply:
good point